sapl/parlamentares/forms.py

```diff
@@ -617,4 +617,55 @@ def save(self, commit=True):
             tipo=tipo,
             nome=bloco.nome
         )
-        return bloco
\ No newline at end of file
+        return bloco
+
+class CargoBlocoForm(ModelForm):
+    class Meta:
+       model = CargoBloco
+       fields = '__all__'
```

Comment:
Indentação incorreta
```suggestion
        fields = '__all__'
```